rocAL/rocAL/include/augmentations/color_augmentations/node_color_twist.h

```diff
@@ -24,28 +24,23 @@ THE SOFTWARE.
 #include "node.h"
 #include "parameter_factory.h"
 #include "parameter_vx.h"
-#include "graph.h"
 
-class ColorTwistBatchNode : public Node
-{
+class ColorTwistNode : public Node {
 public:
-    ColorTwistBatchNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
-    ColorTwistBatchNode() = delete;
+    ColorTwistNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
+    ColorTwistNode() = delete;
     void init(float alpha, float beta, float hue, float sat);
-    void init(FloatParam *alpha, FloatParam *beta, FloatParam *hue, FloatParam *sat);
-
+    void init(FloatParam *alpha_param, FloatParam *beta_param, FloatParam *hue_param, FloatParam *sat_param);
 protected:
     void create_node() override;
     void update_node() override;
 private:
-
     ParameterVX<float> _alpha;
     ParameterVX<float> _beta;
     ParameterVX<float> _hue;
     ParameterVX<float> _sat;
-
-    constexpr static float   ALPHA_RANGE [2] = {0.1, 1.95};
-    constexpr static float   BETA_RANGE [2] = {0.1, 25.0};
-    constexpr static float   HUE_RANGE [2] = {5.0, 170.0};
-    constexpr static float   SAT_RANGE [2] = {0.1, 0.4};
-};
\ No newline at end of file
+    constexpr static float ALPHA_RANGE[2] = {0.1, 1.95};
+    constexpr static float BETA_RANGE[2] = {0.1, 25.0};
+    constexpr static float HUE_RANGE[2] = {5.0, 170.0};
+    constexpr static float SAT_RANGE[2] = {0.1, 0.4};
+};
```

Comment:
Do we not require any blank line at the end of this files?

Reply:
It is present